modules/geo/src/main/java/org/opensearch/geo/search/aggregations/bucket/geogrid/InternalGeoHashGridBucket.java

```diff
@@ -43,7 +43,7 @@
  *
  * @opensearch.internal
  */
-public class InternalGeoHashGridBucket extends InternalGeoGridBucket<InternalGeoHashGridBucket> {
+public class InternalGeoHashGridBucket extends InternalGeoGridBucket {
```

Comment:
Does this change is intentional?

Reply:
it shouldn't be intentional, I might be testing something because of which this change has propagated. I will fix this.